renderdoc/serialise/serialiser.cpp

```diff
@@ -587,15 +587,15 @@ void Serialiser<SerialiserMode::Writing>::WriteStructuredFile(const SDFile &file
 
     Serialiser<SerialiserMode::Writing> *ser = this;
 
+    m_ChunkMetadata.chunkID = 0;
```

Comment:
Please do not make unrelated changes to core serialisation code.

Reply:
Do you think I need to make another PR for that one?

Reply:
Yes please keep each PR related to a single set of functionality, if you have another change you want to propose please make it separately after this one is resolved.